content/applications/marketing/email_marketing.rst

```diff
@@ -16,6 +16,7 @@ successful campaigns, and track their overall effectiveness in a single applicat
 .. seealso::
    - :doc:`/applications/marketing/email_marketing/mailing_lists`
    - :doc:`/applications/marketing/email_marketing/unsubscriptions`
+   - :doc:`/applications/marketing/email_marketing/lost_leads_email`
```

Comment:
Since this file is being edited anyway... please update these to relative paths :)
```suggestion
   - :doc:`email_marketing/mailing_lists`
   - :doc:`email_marketing/unsubscriptions`
   - :doc:`email_marketing/lost_leads_email`
```